src/pcre.mk

```diff
@@ -8,16 +8,9 @@ $(PKG)_VERSION  := 8.45
 $(PKG)_CHECKSUM := 4dae6fdcd2bb0bb6c37b5f97c33c2be954da743985369cddac3546e3218bffb8
 $(PKG)_SUBDIR   := pcre-$($(PKG)_VERSION)
 $(PKG)_FILE     := pcre-$($(PKG)_VERSION).tar.bz2
-$(PKG)_URL      := https://ftp.pcre.org/pub/pcre/$($(PKG)_FILE)
-$(PKG)_URL_2    := https://$(SOURCEFORGE_MIRROR)/project/pcre/pcre/$($(PKG)_VERSION)/$($(PKG)_FILE)
+$(PKG)_URL      := https://$(SOURCEFORGE_MIRROR)/project/pcre/pcre/$($(PKG)_VERSION)/$($(PKG)_FILE)
 $(PKG)_DEPS     := cc
 
-define $(PKG)_UPDATE
-    $(WGET) -q -O- 'https://ftp.pcre.org/pub/pcre/' | \
-    $(SED) -n 's,.*pcre-\([0-9]\+\)\(\.[0-9]\+\)*\.zip.*,\1\2,p' | \
-    tail -1
-endef
-
```

Comment:
According to https://www.pcre.org/, this can be switched to https://github.com/PhilipHazel/pcre2/releases. The sourceforge is an unofficial mirror.

Reply:
I can change that. sure.

Reply:
@tonytheodore Sorry, but this will switch PCRE from v1 to v2 (major release upgrade). That is why we have a `pcre2.mk` file. I will NOT do apply this change!